docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,105 @@
+---
+id: manage-connector-templates
+title: Manage Connector templates
+description: Manage your Connector templates in Web Modeler.
+---
+
+export const UploadIcon = () => <span style={{verticalAlign: "text-top"}}><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16"><path d="M0 0h24v24H0z" fill="none"></path><path d="M9 16h6v-6h4l-7-7-7 7h4zm-4 2h14v2H5z" fill="currentColor"></path></svg></span>;
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+You can create and manage [connector templates](/components/integration-framework/connectors/custom-built-connectors/connector-templates.md) just as any other asset in a Web Modeler project.
+
+## Create connector templates
+
+You can create a new connector template using the following steps:
+
+1. Navigate to the project of your choice and click **New**.
+
+2. Choose to create a blank **Connector Template**.
+
+   ![Creating a new template](img/connector-templates/create-connector-template-1.png)
+
+3. You will be taken to the **connector template editor** screen. In this screen, you can define the connector template by writing the template JSON. The template editor supports you in writing the template by providing autocompletion, error highlighting and a live preview.
+
+   The anatomy of the editor interface is as follows:
+
+   - In the **breadcrumbs bar**, you can rename your template by clicking the chevron next to the template name. Note that you cannot change the name of your template in the template json, but only with this action.
```

Comment:
What is the breadcrumbs bar for clarification given we don't have a screenshot example here?

Reply:
@christinaausley There's a screenshot below, covering all bullet points in this section, but maybe too far away from this text. But I tried to avoid redundancy. I can add one if you like

Reply:
You could always just say "in the **breadcrumbs bar** pictured below,..."

Reply:
You could always just say "in the **breadcrumbs bar** pictured below,..."